hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -1113,7 +1115,7 @@ public void testScmRegisterNodeWith4LayerNetworkTopology()
     try (SCMNodeManager nodeManager = createNodeManager(conf)) {
       DatanodeDetails[] nodes = new DatanodeDetails[nodeCount];
       for (int i = 0; i < nodeCount; i++) {
-        DatanodeDetails node = TestUtils.createDatanodeDetails(
+        DatanodeDetails node = MockDatanodeDetails.createDatanodeDetails(
```

Comment:
NIT: MockDatanodeDetails can be removed as static import is added.

Reply:
The static import was previously only for another method, but added this one as well.